mono/sgen/sgen-alloc.c

```diff
@@ -63,10 +63,19 @@ static guint64 stat_bytes_alloced_los = 0;
 #define TLAB_TEMP_END	(__thread_info__->tlab_temp_end)
 #define TLAB_REAL_END	(__thread_info__->tlab_real_end)
 
+static void 
```

Comment:
Shouldn't this be `static inline` or something like that?

Reply:
Let the compiler decide to inline or not.